### ISSUE TITLE
Too much information send to player

### ISSUE THREAD

**Issue by Kotwic4** [mkaput/secrethitler-server#4]
In answer from mutation there is all information about game. I think we do not use this response, because we only use state from mutations callback

**Comment by mkaput** [comment]
Not sure I'm understanding, but my reasoning was that on client side one could simple swap out client's state with server's response without manually applying mutations themselves

**Comment by Kotwic4** [comment]
Yes client can use only values from registered callback as we do now.

I think we should remove sending whole game config in reponse of user mutation for security. Also clients should not see other users mutations.
